src/Features/Core/Portable/FindUsages/FindUsagesOptions.cs

```diff
@@ -5,17 +5,17 @@
 using System.Runtime.Serialization;
 using Microsoft.CodeAnalysis.Classification;
 
-namespace Microsoft.CodeAnalysis.FindUsages
+namespace Microsoft.CodeAnalysis.FindUsages;
+
+[DataContract]
+internal readonly record struct FindUsagesOptions
 {
-    [DataContract]
-    internal readonly record struct FindUsagesOptions(
-        [property: DataMember(Order = 0)] ClassificationOptions ClassificationOptions)
-    {
-        public FindUsagesOptions()
-            : this(ClassificationOptions.Default)
-        {
-        }
+    [DataMember] public ClassificationOptions ClassificationOptions { get; init; } = ClassificationOptions.Default;
```

Comment:
question: how ar eyou deciding between record-parameter form and record-property form?

Reply:
Switching everything to record-property form to be consistent. Seems like I forgot to convert some options types to this form.